src/mlpack/tests/lars_test.cpp

```diff
@@ -1094,7 +1225,7 @@ TEMPLATE_TEST_CASE("LARSSelectBetaTest", "[LARSTest]", arma::fmat, arma::mat)
   typedef TestType MatType;
   typedef typename MatType::elem_type ElemType;
 
-  const ElemType tol = (std::is_same<ElemType, double>::value) ? 1e-5 : 1e-3;
+  const ElemType tol = (std::is_same<ElemType, double>::value) ? 1e-5 : 5e-3;
```

Comment:
This is a nice way to make the usage of `std::is_same` in a condition with out the need for `constexpr`